src/test/java/uk/gov/companieshouse/pscverificationapi/controller/impl/PscVerificationControllerImplMergeIT.java

```diff
@@ -9,21 +9,14 @@
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
 import static uk.gov.companieshouse.api.model.pscverification.VerificationStatementConstants.INDIVIDUAL_VERIFIED;
-import static uk.gov.companieshouse.api.model.pscverification.VerificationStatementConstants.RO_DECLARATION;
-import static uk.gov.companieshouse.api.model.pscverification.VerificationStatementConstants.RO_IDENTIFIED;
-import static uk.gov.companieshouse.api.model.pscverification.VerificationStatementConstants.RO_VERIFIED;
 
 import java.net.URI;
 import java.time.Clock;
-import java.time.LocalDate;
 import java.time.ZoneId;
 import java.util.EnumSet;
 import java.util.Optional;
 import org.hamcrest.Matchers;
-import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.DisplayName;
-import org.junit.jupiter.api.Tag;
-import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.*;
```

Comment:
IDE probably changed this to a wildcard import. Please fix your IDE settings to prevent this.